src/dataflow/render/context.rs

```diff
@@ -31,10 +31,19 @@ type Diff = isize;
 
 // Local type definition to avoid the horror in signatures.
 pub type Arrangement<S, V> = Arranged<S, TraceValHandle<V, V, <S as ScopeParent>::Timestamp, Diff>>;
+pub type ErrArrangement<S> =
+    Arranged<S, TraceValHandle<EvalError, (), <S as ScopeParent>::Timestamp, Diff>>;
 type ArrangementImport<S, V, T> = Arranged<
     S,
     TraceEnter<TraceFrontier<TraceValHandle<V, V, T, Diff>>, <S as ScopeParent>::Timestamp>,
 >;
+type ErrArrangementImport<S, T> = Arranged<
+    S,
+    TraceEnter<
+        TraceFrontier<TraceValHandle<EvalError, (), T, Diff>>,
```

Comment:
Maybe minor, but if we track down the equivalent of `TraceKeyHandle`, the `()` goes away and the internal representation of the error trace is a bit simpler (one fewer layer in the trie). Probably not a big perf detail, as we don't expect this to be moving lots of error data.

Reply:
Sure thing, done.